apps/api/src/server/repositories/case.repository.js

```diff
@@ -337,6 +340,38 @@ export const updateApplication = ({
 	return databaseConnector.$transaction(transactions);
 };
 
+/**
+ * @param {UpdateApplicationParams} caseInfo
+ * @returns {Promise<PrismaPromise<import('@pins/api').Schema.BatchPayload>>}
+ */
+export const publishCase = async ({ caseId }) => {
+	const transactions = [];
+
+	const application = databaseConnector.case.update({
+		where: { id: caseId },
+		data: {
+			publishedAt: new Date(),
+			modifiedAt: new Date()
+		}
+	});
+
+	transactions.push(application);
+
+	return databaseConnector.$transaction(transactions);
```

Comment:
We don't actually need to use transactions here. Transactions are useful if we have multiple database operations that we would like to process, but here we are only doing a single `update` on `case`

Reply:
ok will change this now